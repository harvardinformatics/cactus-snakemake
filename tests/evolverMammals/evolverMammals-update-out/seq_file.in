(simHuman_chr6:0.144018,(mr:0.171974,simGorilla:0.075)AncGorilla:0.1)Anc1;
mr evolverMammals-update-out/mr.fa
simGorilla evolverMammals-seq/simGorilla.chr6
simHuman_chr6 evolverMammals-update-out/simHuman_chr6.fa
Anc1 evolverMammals-update-out/Anc1.fa
