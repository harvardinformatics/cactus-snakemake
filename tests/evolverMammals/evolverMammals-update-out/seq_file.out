(simHuman_chr6:0.144018,(mr:0.171974,simGorilla:0.075)AncGorilla:0.1)Anc1;
mr	/n/holylfs05/LABS/informatics/Users/gthomas/cactus-snakemake/tests/evolverMammals/evolverMammals-update-out/mr.fa
simGorilla	/n/holylfs05/LABS/informatics/Users/gthomas/cactus-snakemake/tests/evolverMammals/evolverMammals-update-out/simGorilla.chr6
simHuman_chr6	/n/holylfs05/LABS/informatics/Users/gthomas/cactus-snakemake/tests/evolverMammals/evolverMammals-update-out/simHuman_chr6.fa
Anc1	evolverMammals-update-out/Anc1.fa
AncGorilla	/n/holylfs05/LABS/informatics/Users/gthomas/cactus-snakemake/tests/evolverMammals/evolverMammals-update-out/AncGorilla.fa
